epochStart/metachain/stakingDataProvider.go

```diff
@@ -18,25 +18,31 @@ import (
 )
 
 type ownerStats struct {
-	numEligible        int
-	numStakedNodes     int64
-	topUpValue         *big.Int
-	totalStaked        *big.Int
-	eligibleBaseStake  *big.Int
-	eligibleTopUpStake *big.Int
-	topUpPerNode       *big.Int
-	blsKeys            [][]byte
+	numEligible          int
+	numStakedNodes       int64
+	numActiveNodes       int64
+	numAuctionNodes      int64
+	totalTopUp           *big.Int
+	topUpPerNode         *big.Int
+	totalStaked          *big.Int
+	eligibleBaseStake    *big.Int
+	eligibleTopUpStake   *big.Int
+	eligibleTopUpPerNode *big.Int
+	blsKeys              [][]byte
+	auctionList          []state.ValidatorInfoHandler
+	qualified            bool
 }
 
 type stakingDataProvider struct {
-	mutStakingData          sync.RWMutex
-	cache                   map[string]*ownerStats
-	systemVM                vmcommon.VMExecutionHandler
-	totalEligibleStake      *big.Int
-	totalEligibleTopUpStake *big.Int
-	minNodePrice            *big.Int
-	stakingV4EnableEpoch    uint32
-	flagStakingV4Enable     atomic.Flag
+	mutStakingData             sync.RWMutex
+	cache                      map[string]*ownerStats
+	numOfValidatorsInCurrEpoch uint32
```

Comment:
`numOfValidatorsInCurrEpoch` can  be moved near `stakingV4EnableEpoch` in order to group variables of the same type

Reply:
Refactored as suggested